lib/chef/telemetry.rb

```diff
@@ -0,0 +1,76 @@
+#
+# Author:: Ranjib Dey (<ranjib@linux.com>)
+# Copyright:: Copyright (c) 2015 Chef Inc.
```

Comment:
There is no such company as "Chef Inc", it is "Chef Software, Inc.". </legal pedant>